include/openssl/err.h

```diff
@@ -301,7 +303,7 @@ int ERR_load_ERR_strings(void);
 
 DEPRECATEDIN_1_1_0(void ERR_remove_thread_state(void *))
 DEPRECATEDIN_1_0_0(void ERR_remove_state(unsigned long pid))
-ERR_STATE *ERR_get_state(void);
+DEPRECATEDIN_3(ERR_STATE *ERR_get_state(void))
```

Comment:
Seems like this deserves a CHANGES entry

Reply:
Yes, thanks for the reminder